package.json

```diff
@@ -2,9 +2,12 @@
   "name": "core-decorators",
   "version": "0.0.13",
   "description": "Library of ES7 decorators inspired by languages that come with built-ins like @override, @deprecated, etc",
-  "main": "src/core-decorators.js",
+  "main": "lib/core-decorators.js",
+  "files": "lib",
   "scripts": {
-    "test": "mocha --require babel/register 'src/**/*.spec.js'"
+    "test": "mocha --require babel/register 'src/**/*.spec.js'",
+    "build": "babel --stage 0 --out-dir lib src",
+    "prepublish": "npm run-script build"
```

Comment:
I know right...this has bitten me several times over the years (I keep forgetting) so I made it a policy _never_ to use it _ever_. Thanks!